packages/autorest.python/autorest/preprocess/__init__.py

```diff
@@ -267,23 +267,15 @@ def update_types(self, yaml_data: List[Dict[str, Any]]) -> None:
                     type.get("description", ""), type["name"]
                 )
                 type["snakeCaseName"] = to_snake_case(type["name"])
-            if type.get("values") and not self.version_tolerant:
+            if type.get("values"):
                 # we're enums
-                values_to_add = []
                 for value in type["values"]:
                     padded_name = self.pad_reserved_words(
                         value["name"].lower(), PadType.ENUM
                     ).upper()
-                    if value["name"] != padded_name:
-                        values_to_add.append(
-                            update_enum_value(
-                                name=padded_name,
-                                value=value["value"],
-                                description=value["description"],
-                                enum_type=value["enumType"],
-                            )
-                        )
-                type["values"].extend(values_to_add)
+                    if padded_name[0] in "0123456789":
```

Comment:
nit: `isdigit()`: https://www.w3schools.com/python/ref_string_isdigit.asp

Reply:
I don't need to check string, only first letter. The code is more efficient.